examples/azure-identity/go/windows.Dockerfile

```diff
@@ -0,0 +1,29 @@
+ARG BUILDER=mcr.microsoft.com/oss/go/microsoft/golang:1.19-bullseye
+ARG SERVERCORE_CACHE=gcr.io/k8s-staging-e2e-test-images/windows-servercore-cache:1.0-linux-amd64-${OS_VERSION:-1809}
```

Comment:
do we need this?

Reply:
yeah, that is required for the `netapi32.dll`. The network calls fail without it.